### DISCUSSION TITLE
github for students

### DISCUSSION THREAD

**Discussion by Hawkazizi** [community/community#134427] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Profile

### Body

hello dear github users ,I  just heard about github for students and its benefits for us , but when i try to add my college email to my profile, it gives me this error: 
 Error adding shahin.azizi@ut.ac.ir: Email may be for a government official. Please see the FAQ at https://docs.github.com/en/github/site-policy/github-and-trade-controls for more information.

p.s I am from Iran and it is Tehran university, any ideas on how to solve this , or is it even possible to contact with github supporters directly?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Hawkazizi ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
